code/game/machinery/posialert_console.dm

```diff
@@ -0,0 +1,60 @@
+/*************************
+*** Posi-Alert Console ***
+*************************/
+
+/obj/machinery/posialert_console
```

Comment:
any reason to not make this a computer? no way to construct or deconstruct either...

Reply:
as others have said, its really a small function to be made into a computer.
Ill add a way to construct, deconstruct may be a little harder.